corehq/apps/app_manager/detail_screen.py

```diff
@@ -352,8 +352,39 @@ def variables(self):
         return variables
 
 
+@register_format_type('conditional-enum')
+class ConditionalEnum(Enum):
+    @property
+    def sort_node(self):
+        node = super(ConditionalEnum, self).sort_node
+        if node:
+            node.order = "-2"
+            node.direction = "ascending"
```

Comment:
I don't think that's correct re the sort order etc. If you look at this you'll see that `-2` means index: https://github.com/dimagi/commcare-hq/blob/f390477b6ac24c0c0321d915201abf5dbf820161/corehq/apps/app_manager/detail_screen.py#L175-L175